newdle/config.py

```diff
@@ -0,0 +1,12 @@
+from flask import Blueprint, current_app
+
+
+config = Blueprint('config', __name__, url_prefix='/config')
```

Comment:
I don't think we need a new file/blueprint for this; this is just another API after all...